platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/VirtualHashRecordSerializer.java

```diff
@@ -50,45 +62,80 @@ public long getCurrentDataVersion() {
     }
 
     @Override
-    public int getHeaderSize() {
-        return Long.BYTES; // path
+    // TODO: is it really used anywhere?
```

Comment:
Either delete it or create a ticket, please.

Reply:
I'll remove the constant. For now, virtual hash records will be of variable size (no single size can be returned from getSerializedSize() to match both JDB and PBJ). After JDB support is dropped, the records can be changed to fixed size again.